src/redist/targets/BundledTemplates.targets

```diff
@@ -1,4 +1,14 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="CalculateTemplatesVersion"
+          DependsOnTargets="SetSdkVersionInfo">
+    <PropertyGroup>
+      <MicrosoftNETCoreAppVersionPreReleaseSeparator>$(MicrosoftNETCoreAppPackageVersion.IndexOf('-'))</MicrosoftNETCoreAppVersionPreReleaseSeparator>
+      <MicrosoftNETCoreAppMajorMinorPatchVersion>$(MicrosoftNETCoreAppPackageVersion)</MicrosoftNETCoreAppMajorMinorPatchVersion>
+      <MicrosoftNETCoreAppMajorMinorPatchVersion Condition=" '$(MicrosoftNETCoreAppVersionPreReleaseSeparator)' != -1 ">$(MicrosoftNETCoreAppPackageVersion.Substring(0, $(MicrosoftNETCoreAppVersionPreReleaseSeparator)))</MicrosoftNETCoreAppMajorMinorPatchVersion>
+      <BundledTemplatesVersion>$(MicrosoftNETCoreAppMajorMinorPatchVersion)-$(VersionSuffix)</BundledTemplatesVersion>
```

Comment:
Should this always have a '-'? What about when we're building a final build?

Reply:
Great catch. @johnbeisner can you address that?